### ISSUE TITLE
help

### ISSUE THREAD

**Issue by jaan143** [TheClashster/LinkCheck#6]
is your project working still ?
i am trying to use but i got error this one (ERROR: Not a valid URL)

i add url at here
raw_url = input("Input the URL where you want to find the link: https://www.udemy.com/course/60-min-website-mobile-optimisation-and-seo-in-wix")
pattern = input("Input the pattern which you want to find:Buy now ")

please correct me if i am wrong
thank you

**Comment by aadibajpai** [comment]
> is your project working still ?
> i am trying to use but i got error this one (ERROR: Not a valid URL)
> 
> i add url at here
> raw_url = input("Input the URL where you want to find the link: https://www.udemy.com/course/60-min-website-mobile-optimisation-and-seo-in-wix")
> pattern = input("Input the pattern which you want to find:Buy now ")
> 
> please correct me if i am wrong
> thank you

yeah it should work, but it's optimized for searching for links so just the text "Buy now" would probably not work unless it was in a link

**Comment by heraldofsolace** [comment]
Are you by any chance editing the source file and add your URL in there? You're supposed to run the file in the terminal. Check the usage section in README
